ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Common/tasks/RedHat.yml

```diff
@@ -95,11 +95,12 @@
     - ansible_architecture == "ppc64"
   tags: build_tools
 
-- name: Install additional build tools for RHEL on s390x
+- name: Install additional build tools for RHEL7 on s390x (31-bit support)
   package: "name={{ item }} state=latest"
-  with_items: "{{ Additional_Build_Tools_RHEL_s390x }}"
+  with_items: "{{ Additional_Build_Tools_RHEL7_s390x }}"
   when:
     - ansible_architecture == "s390x"
+    - (ansible_distribution_major_version == "7")
```

Comment:
I think the more correct solution would be when distribution is <= 7. I'm not sure what the syntax would be for that or if something like that is supported. We added these packages for rhel6 so we'd at least need "when 6 || 7"

Reply:
Happy to do that change - Adoptium has never had RHEL6 so that's not something we could test or would use.